### DISCUSSION TITLE
Upload to ESP32 Single Core

### DISCUSSION THREAD

**Discussion by markuskooche** [HomeSpan/HomeSpan#399] [Q&A]
Currently I am working on a light switch. For this I need a device with an ESP32-U4WDH (Single core, 160MHz, 4MB embedded flash). Unfortunately I can't flash anything to the device via Arduino (neither Ardunio nor Ardunio IDE). Has anyone had any experience with this?

**Comment by HomeSpan** [comment]
What is the manufacturer of the board?  Does it show up in the Arduino IDE as an option?

**Reply by markuskooche** [reply]
Unfortunately, I do not know the manufacturer of the board.
The board is installed in a Shelly Plus 1.

No unfortunately it is not displayed option.
In the meantime I have switched to a different product. (Sonoff DUAL R3)
A dual core is installed here. I hope that I then get it to work.
